### ISSUE TITLE
Always allow column 1 comments?

### ISSUE THREAD

**Issue by jeffreykegler** [jeffreykegler/yahc#39]
I think it may be a good idea to always allow column 1 comments.  These are used in 181 places, for things like commenting out code, meta-remarks, etc.  This seems like an eminently reasonable thing for the coders to be doing.

Here is one example, from `app/gmail.hoon`:

```
  ::  Subscription request
::    ++  listen
::      ^+  .
::      =+  events=?>(?=([@ @ *] pax) t.t.pax)
::      |-  ^+  +>.$
::      ?~  events
::        +>.$
::      ?:  (~(has by web-hooks) i.events)                ::  if hook exists
::        =.  +>.$  (update-hook i.events)
::        $(events t.events)
::      =.  +>.$  (create-hook i.events)
::      $(events t.events)
    ::
 
```

@ohAitch -- do you agree?

**Comment by ohAitch** [comment]
Yup, I think a bunch of editors have a "block line comment" function that has been employed, including I suppose vim; seems like a good exception to have.

**Comment by jeffreykegler** [comment]
Fixed in commit a1fadc4.  Marking "ready to be closed"

**Comment by jeffreykegler** [comment]
Closed as per the above.
